packages/components/text/src/text.vue

```diff
@@ -1,11 +1,25 @@
 <template>
-  <component :is="tag" :class="textKls">
-    <slot />
-  </component>
+  <el-tooltip v-bind="tooltipOptions">
+    <component
+      :is="tag"
+      ref="textRef"
+      :class="textKls"
+      :style="{ '-webkit-line-clamp': lineClamp }"
+      v-bind="$attrs"
+    >
+      <slot />
+    </component>
+    <template #content>
+      <slot v-if="$slots.tooltip" name="tooltip" />
+      <slot v-else />
```

Comment:
## Question

```vue
<slot v-else />
```

What case does this part apply to?

Reply:
If no use tooltip slot, else use default slot, show in tooltip content.